hbase-server/src/test/java/org/apache/hadoop/hbase/util/TestEncryptionTest.java

```diff
@@ -91,6 +92,35 @@ public void testTestCipher() {
     } catch (Exception e) { }
   }
 
+  @Test
+  public void testTestEnabled() {
```

Comment:
follow on:

* break this into two tests e.g. `testDefaultEnabled` and `testSetToEnabled`.
* the third test condition feels like it is duplicating work done in `TestEncryptionDisabled`, but if I'm wrong about that then make it a third test an use JUnit's expected for the exception.